libsquoosh/src/codecs.ts

```diff
@@ -1,5 +1,28 @@
 import { promises as fsp } from 'fs';
-import { instantiateEmscriptenWasm, pathify } from './emscripten-utils.js';
+import { instantiateEmscriptenWasm, pathify } from './emscripten-utils';
+
+interface RotateModuleInstance {
+  exports: {
+    memory: WebAssembly.Memory;
+    rotate(width: number, height: number, rotate: number): void;
+  };
+}
```

Comment:
We should de-dupe that with the definition in `src/features/preprocessors/rotate/worker/rotate.ts` at some point. But we can do that in a separate PR.

Reply:
Makes sense, where do you think we should add the type definitions; would `codecs/rotate` work?

Reply:
I think we should just have `codecs/rotate/rotate.ts` that wraps Wasm, yeah.

Reply:
Awesome! I'll do it in a separate PR :)